pom.xml

```diff
@@ -53,12 +53,17 @@
     <revision>2.8</revision>
     <changelist>-SNAPSHOT</changelist>
     <java.level>8</java.level>
-    <workflow-support.version>2.12</workflow-support.version>
-    <jenkins.version>2.60.3</jenkins.version>
+    <workflow-support.version>3.4</workflow-support.version>
+    <jenkins.version>2.138.4</jenkins.version>
```

Comment:
Looking at https://stats.jenkins.io/pluginversions/lockable-resources.html we see that
- people who updated in the last year (at least version 2.5): 161,473
- from those we loose (lower then 2.138.4): 413
That's about 0.26% - I'm okay with that new baseline :grin:

Reply:
Great!